sound/soc/sof/intel/hda-codec.c

```diff
@@ -117,6 +117,66 @@ int hda_codec_probe_bus(struct snd_sof_dev *sdev)
 }
 EXPORT_SYMBOL(hda_codec_probe_bus);
 
+static void hda_pci_update_bits(struct pci_dev *pci, u32 offset,
+				u32 mask, u32 value)
+{
+	u32 new, ret;
+
+	pci_read_config_dword(pci, offset, &ret);
+	new = (ret & (~mask)) | (value & mask);
+	pci_write_config_dword(pci, offset, new);
+}
```

Comment:
don't we have a helper for this already?

Reply:
yes, I will refine it